docs/proposals/20191030-machine-health-checking.md

```diff
@@ -105,14 +104,12 @@ As an operator of a Management Cluster, I want my machines to be self-healing an
 
 ### Implementation Details/Notes/Constraints
 
-#### Machine annotation:
+#### Machine condition:
 ```go
-const MachineUnhealthyAnnotation = "machine.cluster.x-k8s.io/unhealthy"
+const ConditionHealthCheckSucceeded ConditionType = "HealthCheckSucceeded"
```

Comment:
I'm definitely not good at naming, but since this condition represent an ongoing status, this should be end in `Ready`. What about “HealthzReady“?

Reply:
Hmm, I'm not sure I get what you mean. To me this condition represents that the last health check we ran succeeded. Nothing is "ready" as a result.